src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/ClassDataContract.cs

```diff
@@ -1458,94 +1252,223 @@ internal bool IsScriptObject
                 return ctor;
             }
 
-            internal XmlFormatClassWriterDelegate? XmlFormatWriterDelegate
-            {
-                get { return _xmlFormatWriterDelegate; }
-                set { _xmlFormatWriterDelegate = value; }
-            }
+            internal XmlFormatClassWriterDelegate? XmlFormatWriterDelegate { get; set; }
 
-            internal XmlFormatClassReaderDelegate? XmlFormatReaderDelegate
-            {
-                get { return _xmlFormatReaderDelegate; }
-                set { _xmlFormatReaderDelegate = value; }
-            }
+            internal XmlFormatClassReaderDelegate? XmlFormatReaderDelegate { get; set; }
 
-            public XmlDictionaryString?[]? ChildElementNamespaces
-            {
-                get { return _childElementNamespaces; }
-                set { _childElementNamespaces = value; }
-            }
+            internal XmlDictionaryString?[]? ChildElementNamespaces { get; set; }
 
             private static Type[] SerInfoCtorArgs => s_serInfoCtorArgs ??= new Type[] { typeof(SerializationInfo), typeof(StreamingContext) };
 
             internal struct Member
             {
                 internal Member(DataMember member, string ns, int baseTypeIndex)
                 {
-                    this.member = member;
-                    this.ns = ns;
-                    this.baseTypeIndex = baseTypeIndex;
+                    _member = member;
+                    _ns = ns;
+                    _baseTypeIndex = baseTypeIndex;
                 }
-                internal DataMember member;
-                internal string ns;
-                internal int baseTypeIndex;
+                internal DataMember _member;
+                internal string _ns;
```

Comment:
What's the naming convention for non-private fields? To me it always feels wrong to write otherThing._field from the non owning class.

Reply:
https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md
Point 3 says to use '_'